src/utils.ts

```diff
@@ -614,6 +614,58 @@ async function verifyFlavorPath(flavor: BrowserFlavor | undefined, platform: Pla
     }
 }
 
+type ExtensionSettings = [string, boolean | string | {[key: string]: string} | undefined];
+
+export function reportExtensionSettings(telemetryReporter: Readonly<TelemetryReporter>): void {
+    const extensionSettingsList = Object.entries(vscode.workspace.getConfiguration(SETTINGS_STORE_NAME)).splice(4) as Array<ExtensionSettings>;
```

Comment:
This is the best way I could get a list of all the extension settings.  The `workspaceConfiguration` object has getters, but nothing to return an iterable object.  However, the `workspaceConfiguration` object contains each setting as a property.  I splice it at the 4th index since the first 4 properties are all functions related to the `workspaceConfiguration` class and the rest are the extension properties under `SETTINGS_STORE_NAME`.

`extensionSettingsList` ends up looking something like:
```
[
  ["enableNetwork", "false"],
  ["themes", "Light"],
  ["screencastCursorType", "Default"],
]
...
```